src/Common/src/System/Net/Logging/GlobalLog.cs

```diff
@@ -253,12 +253,17 @@ public static void Dump(byte[] buffer, int offset, int length)
                 return;
             }
 
+            if (length == 0)
+            {
+                EventSourceLogging.Log.DebugMessage("lenth of ArraySegment in Dump is 0");
```

Comment:
To keep the trace uniform, instead of creating a different type of non-warning message, could we call `DebugDumpArray(Array.Empty<byte>())`?